shared/src/models/utils/tag-helpers.ts

```diff
@@ -134,3 +136,80 @@ export function createTreatmentProgressTag(
         treatmentProgress
     );
 }
+
+export function createVehicleTag(
+    draftState: Mutable<ExerciseState>,
+    vehicleId: UUID
+): Tag {
+    const vehicle = getElement(draftState, 'vehicle', vehicleId);
+    return new Tag('Fahrzeug', 'grey', 'white', vehicle.name, vehicleId);
+}
+
+export function createVehicleTypeTag(
+    draftState: Mutable<ExerciseState>,
+    vehicleId: UUID
+): Tag {
+    const vehicle = getElement(draftState, 'vehicle', vehicleId);
+    return new Tag(
+        'Fahrzeugtyp',
+        'grey',
+        'white',
+        vehicle.vehicleType,
+        vehicle.vehicleType
+    );
+}
+
+export function createOccupationTag(
+    _draftState: Mutable<ExerciseState>,
+    occupation: ExerciseOccupation
+): Tag {
+    return new Tag(
+        'Tätigkeit',
+        'white',
+        'black',
+        occupationToGermanDictionary[occupation.type],
+        occupation.type
+    );
+}
+
+export function createVehicleActionTag(
+    _draftState: Mutable<ExerciseState>,
+    vehicleAction: 'arrived' | 'departed' | 'load' | 'unload'
```

Comment:
Could you plase normalize the strings so thex're all in the same tense (e.g. `loaded` and `unloaded`)?